_events/2020-03-13-meetup.md

```diff
@@ -30,17 +30,31 @@ Please register via this [Eventbrite page](https://www.eventbrite.co.uk/e/nl-rse
 | 11:00 - 11:05 | Mateusz Kuzak | | Welcome and purpose |
 | 11:05 - 11:10 | Mateusz Kuzak | | Agenda and ground rules | 
 | 11:10 - 11:20 | everyone | | Introductions and announcements |
-| 11:20 - 11:50 | Gijs Molenaar| talk | [http://pythonic.nl](http://pythonic.nl),  tba|
+| 11:20 - 11:50 | Gijs Molenaar| talk | "Training a reinforcement agent with simulations of the Dutch electricity Grid to optimize long-term investment strategies" [http://pythonic.nl](http://pythonic.nl)|
 | 11:50 - 12:20 | Emmy Tsang | talk | "Community driven solutions for better research communication", [eLife Innovation](https://elifesciences.org/labs)|
 | 12:20 - 12:35 | anyone | talk | Lightning Talks |
 | 12:35 - 13:30 | | | Lunch |
-| 13:30 - 14:20 | Felipe Zapata | hands-on | [Continuous Integration and Continuous developement for scientific applications](https://github.com/NLESC-JCER/ci_for_science) |
+| 13:30 - 14:20 | Felipe Zapata | hands-on | "Continuous integration and Continuous Delivery for scientific applications" [ci for science](https://github.com/NLESC-JCER/ci_for_science) |
 | 14:20 - 14:30 | | | Tea |
 | 14:30 - 15:20 | everyone | hands-on | What makes good documentation? Interactive session |
 | 15:20 - 15:25 | | discussion | Parking Lot |
 | 15:25 - 15:30 | | discussion | Evaluation |
 | 15:30 - 16:00 | | | Drinks |
 
+## Abstracts
+
+### Gijs Molenaar "Training a reinforcement agent with simulations of the Dutch electricity Grid to optimize long-term investment strategies"
+As a free-lance research engineer, I'm currently involved in creating a simulation of the Dutch electricity grid maintained by Alliander. These simulations run 40 years in the future, repeated numerous times each with a different set of random growth conditions. The outputs of these simulations are used to train a deep reinforcement agent the optimise long-term investment strategies of the company.
+
+In this talk, I will talk about the tools and techniques used (Python, NumPy, Kubernetes, Neo4j), our development infrastructure and our team's work distribution.
+
+### Emmy Tsang "Community driven solutions for better research communication"
+
```

Comment:
Can we add some text here? "Abstract pending" or something, just so the section is not empty?